docs/DeveloperGuide.md

```diff
@@ -401,6 +401,31 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 * Steps 2b1-2b2 are repeated until all details entered are correct.
 * Use cases resumes from step 3.
 
+**Use case: UC07 - Settle outstanding fees for student**
+
+**MSS**
+
+1. User requests to <ins>list students(UC02)</ins>.
+2. User enters command to settle outstanding fees for the specified student.
+3. System updates the total tuition fee paid and total tuition fee owed by the student.
+4. System displays success message.
+
+   Use case ends.
+
+**Extensions**
+
+* 2a. The given index is invalid.
```

Comment:
Can i check if using the word index is implementation specific? Furthermore, wouldn't the next extension catch this error as well?

Reply:
I am not sure too. The other use cases also seem to use index. As for the next extension catching the error, I am unsure to whether lump all the errors together or to separate by types of errors

Reply:
I have removed the extension which uses index